### PR TITLE
fix: operator translate exposeMonitoring for scanner v4

### PR DESCRIPTION
**Pull request by johannes94**
## Description

Allow to configure exposeMonitoring helm value for scanner v4 CRDs.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: operator/apis/platform/v1alpha1/securedcluster_types.go [hunk 1]
```diff
@@ -370,6 +370,12 @@ type LocalScannerV4ComponentSpec struct {
 	// Settings pertaining to the DB deployment.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldDependency:.scannerComponent:AutoSense"}
 	DB *ScannerV4DB `json:"db,omitempty"`
+
+	// Configures monitoring endpoint for Scanner V4. The monitoring endpoint
+	// allows other services to collect metrics from Scanner V4, provided in
+	// Prometheus compatible format.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
+	Monitoring *Monitoring `json:"monitoring,omitempty"`
```
Comment (vladbologa) [inline operator/apis/platform/v1alpha1/securedcluster_types.go:378]: I'm wondering if there's a reason this wasn't exposed in `LocalScannerComponentSpec` above (in secured clusters). @stehessel could you provide some context here?

**Review by vladbologa — APPROVED** [review]

**Comment by johannes94** [comment]
/retest

Looks like a timeout on waiting for a ready `scanner-v4-db` pod but at the time of pod describes it is ready.

**Pull request merged**